### DISCUSSION TITLE
Error while pulling the code from git hub to workstation

### DISCUSSION THREAD

**Discussion by bharath-gaveni** [learndevopsonline/learndevopsonline#1603] [Q&A] [answered]
Error while pulling the code from git hub to workstation 

![Screenshot (5)](https://github.com/learndevopsonline/learndevopsonline/assets/149563378/d2744726-891b-4772-9211-2c318c78fdca)

**Comment by sreedharm07** [comment, marked as answer]
Hi @bharath-gaveni 
just give **git stash**  if next time you face this type of  error.
for now i did in your machine. you can pull now 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/ff0c5193-6271-4570-bdb1-b6121e8614fa)

**Comment by bharath-gaveni** [comment]
okay @sreedharm07  thank you
